modules/ROOT/pages/index.adoc

```diff
@@ -2,9 +2,15 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:linkattrs:
 
-Use Anypoint Platform to build a structured application network that connects applications, data, and devices with reusable APIs.
+Your customers and employees need data-rich, delightful digital experiences on a variety of devices from smart watches to desktop computers. To deliver these experiences, your systems must be connected to each other, and the data must flow among those systems (integration).
+
+The MuleSoft approach to integration, https://blogs.mulesoft.com/dev/api-dev/what-is-api-led-connectivity/[integration of data from different systems using a layer of APIs], allows you to spend less time on IT projects and more time on your core business. Whenever you turn a business process into an API, you make it easier to implement that process in the next project. And APIs have proven to be the best size of reusable code. 
+
+image::api-led-archicture.png[]
```

Comment:
@fermujica or @sleslie23 this is the image I added.

Reply:
I don't see any image named api-led-archicture.png in this repo.

Reply:
fixed.

Reply:
Fer confirmed the image appears in output.